ska_conda/pkg_defs/ska-flight/meta.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: ska-flight
+  version: 0.1
+
+requirements:
+  # Packages required to run the package. These are the dependencies that
+  # will be installed automatically whenever the package is installed.
+  run:
+    - ska.shell ==3.3.3
```

Comment:
Alphabetize for convenience in finding a package to update version.

Reply:
Good point.  We may end up doing this in other packages too, as largely they just accumulated dependencies as they were considered.